README.md

```diff
@@ -63,7 +63,7 @@ If you're feeling lucky, go ahead ;). Copying characters is error prone and migh
 
 ## Contributing
 
-You can refer to the Unicode Character table site for integrating more characters: [http://unicode-table.com/en](http://unicode-table.com/en) and [http://jrgraphix.net/r/Unicode/](http://jrgraphix.net/r/Unicode/).
+You can refer to the Unicode Character table site for integrating more characters: [https://symbl.cc/en](https://symbl.cc/en) and [http://jrgraphix.net/r/Unicode/](http://jrgraphix.net/r/Unicode/).
```

Comment:
Should we link to the table directly? 

```suggestion
You can refer to the Unicode Character table site for integrating more characters: [https://symbl.cc/en/unicode/table/](https://symbl.cc/en/unicode/table/) and [http://jrgraphix.net/r/Unicode/](http://jrgraphix.net/r/Unicode/).
```

Reply:
> Should we link to the table directly?

Hello. There is no need to link to a specific page with a symbol table, because initially the URL pointed to the start page of the Unicode table site. Although your version looks correct. I leave it to your discretion, there is no mistake here. Thanks for the question)